src/Json.Converter.fs

```diff
@@ -614,7 +614,7 @@ module Convert =
             let unknownType = getType()
             failwithf "Cannot convert %s to %s" (SimpleJson.toString input) unknownType.FullName
         | _ ->
-            failwithf "Cannot convert %s to %s" (SimpleJson.toString input) (JS.JSON.stringify typeInfo)
```

Comment:
Cannot use `json.dumps` here since it don't know how to serialize the class `TypeInfo`. I also tried `Convert.serialize typeInfo typeInfo` but that gave some other error that I could not figure out:

```
    def Convert_betweenQuotes(input: str) -> str:
>       return ("\"" + input) + "\""
E       TypeError: can only concatenate str (not "TypeInfo") to str
```